kpatch-build/create-diff-object.c

```diff
@@ -771,6 +771,88 @@ static bool kpatch_line_macro_change_only(struct section *sec)
 
 	return true;
 }
+
+#elif __s390x__
+
+static int _insn_get_length(unsigned char *inst)
```

Comment:
If I understand correctly the only reason you are prefixing this function with a `_` is that it is already defined in `insn/asm/insn.h`. We only build insn dir for x86_64 so I think that define can be safely moved under `#ifdef __x86_64__` and then we'll be able to use common `insn_get_length` name.

Reply:
+1